test/integration/WarmResumptionTest.cpp

```diff
@@ -66,7 +66,7 @@ TEST_F(ServerFixture, DISABLED_BasicWarmResumption) {
   clientEvb->runInEventBaseThreadAndWait([&]() { rsocket->disconnect(); });
 
   // This request should be buffered
-  mySub->request(2);
+  clientEvb->runInEventBaseThreadAndWait([&]() { mySub->request(2); });
```

Comment:
Thanks for fixing this. I will have a class which helps with the threading model and will test it here.